integration-tests/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 quarkus.http.port=8081
+quarkus.temporal.context-propagator-classes[0]=io.quarkiverse.temporal.it.util.MDCContextPropagator
```

Comment:
Example usage of plugging in your own context propagators

Reply:
shouldn't the built in propagators be added by default  ?

Reply:
Are there built in propagators?  This was all done based on this Spring Boot Example https://github.com/Quinn-With-Two-Ns/samples-java/commit/c99e24af5f20d1965827c5e178f6541a9b5ddf12#diff-110913636fdd14fbceea63df5e4a32e6815c72549d7420cea869c3f8455fc886